### ISSUE TITLE
Settings for Mask RCNN version

### ISSUE THREAD

**Issue by kenoharada** [vimalabs/VIMA#1]
Thank you for sharing your work! I really enjoy playing with VIMA.

Your code in example.py assumes ground truth segmentation mask of object is given.

Can you share codes for Mask RCNN version where segmentation mask is not given by simulator?
Or can you tell me which kind of modification is needed for Mask RCNN version?

**Comment by yunfanjiang** [comment]
Hey, thanks for your interest! We are currently working on clean-up and will release corresponding code soon.

**Comment by kenoharada** [comment]
Thank you!

**Comment by kenoharada** [comment]
@yunfanjiang training code is going to be public in the future?

**Comment by yunfanjiang** [comment]
Because the training code is tightly coupled with company infra, I'm afraid that we will not be able to disclose it. Apologize for that.
